include/retry.h

```diff
@@ -78,27 +78,39 @@ namespace azure {  namespace storage_lite {
     class retry_policy final : public retry_policy_base
     {
     public:
+
+        retry_policy(const int maximum_retries = 3, const std::chrono::seconds base_timeout = std::chrono::seconds{10})
```

Comment:
Please make the `maximum_retries` and `base_timeout` a reference to a constant variable defined in constant.dat.
This way it is easier for the customer to find where to change them by modifying the code.

Reply:
Done, please re-check